filebeat/input/stdin/input.go

```diff
@@ -5,38 +5,38 @@ import (
 
 	"github.com/elastic/beats/filebeat/channel"
 	"github.com/elastic/beats/filebeat/harvester"
+	"github.com/elastic/beats/filebeat/input"
 	"github.com/elastic/beats/filebeat/input/file"
 	"github.com/elastic/beats/filebeat/input/log"
-	"github.com/elastic/beats/filebeat/prospector"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
 func init() {
-	err := prospector.Register("stdin", NewProspector)
+	err := input.Register("stdin", NewInput)
 	if err != nil {
 		panic(err)
 	}
 }
 
-// Prospector is a prospector for stdin
-type Prospector struct {
+// Input is an input for stdin
+type Input struct {
```

Comment:
is Input used in any other place? Let's not export symbols not required by other modules.

E.g. plugin types in libbeat outputs are not exported.

Reply:
I don't think its used outside like I've said when I have renamed the `Prospector/input`, I didn't revise any visibility I can certainly do a check on it, since all those PRs are small.

Reply:
Let's do this in a second step as it will cause name conflicts in this PR.

I'm tempted to say there will be Inputs which use some part of an other Input.

Reply:
After giving some thinking I agree with @ruflin